Sources/TuistKit/Commands/CommandRegistry.swift

```diff
@@ -120,12 +121,13 @@ public final class CommandRegistry {
         return hiddenCommands[commandName]
     }
 
-    private func process(arguments: ArgumentParser.Result) throws {
+    private func process(arguments: ArgumentParser.Result, parser: ArgumentParser) throws {
         guard let subparser = arguments.subparser(parser) else {
             parser.printUsage(on: stdoutStream)
             return
         }
-        if let command = commands.first(where: { type(of: $0).command == subparser }) {
+        let allCommands = commands + commands.flatMap { $0.subcommands }
+        if let command = allCommands.first(where: { type(of: $0).command == subparser }) {
```

Comment:
Why do we need to ask all commands for their subcommand?

Reply:
I did not want to check here just for `SigningCommand`, though it would be possible